pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2359,6 +2359,7 @@ def buildScriptsAssemble(
                     context.stage("${testTarget}") {
                         setStageResult("${testTarget}", remoteJobStatus)
                     }
+                    context.println "Build ${testTarget} completed: ${remoteJobStatus}"
```

Comment:
I'd like to mimic the same console output as the test pipelines use, so I can re-use the same code in TRSS to find and report status of these remote triggered jobs.